decentraland/unity-explorer
# fix: count command chat messages

## What does this PR change?

All sent chat messages considered to be same on Analytics, so I added property to have possibility to filter chat commands on the backend

## How to test the changes?

You cannot verify adjustment to analytics, but it would be good to check that these variations of /goto are working fine:
1. from JumpIn button on Navmap
2. from Back-to-Genesis on Minimap
3. from chat `/goto`

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
